envFiles/.env.deploy

```
@@ -113,4 +113,11 @@ API_OTEL_ENABLED=true # make it "false" if don't want span logging
 API_OTEL_SERVICE_NAME="talawa-api"
 API_OTEL_ENVIRONMENT="production"
 API_OTEL_SAMPLING_RATIO="1"
-API_OTEL_EXPORTER_OTLP_ENDPOINT="http://localhost:4318/v1/traces" #TODO: after integrating Observability tool
\ No newline at end of file
+API_OTEL_EXPORTER_OTLP_ENDPOINT="http://localhost:4318/v1/traces" #TODO: after integrating Observability tool
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_OTEL_EXPORTER_OTLP_ENDPOINT key should go before the API_OTEL_SAMPLING_RATIO key

(UnorderedKey)